crates/build/src/util/mod.rs

```diff
@@ -104,9 +104,9 @@ pub fn cargo_tty_output(cmd: Expression) -> Expression {
         .map(|(width, _)| width.to_string())
         .unwrap_or_else(|| "100".to_string());
 
-    cmd.env("CARGO_TERM_COLOR", "always")
+    cmd.env("CARGO_TERM_COLOR", "auto")
         .env("CARGO_TERM_PROGRESS_WIDTH", term_size)
-        .env("CARGO_TERM_PROGRESS_WHEN", "always")
+        .env("CARGO_TERM_PROGRESS_WHEN", "auto")
```

Comment:
Interesting that this actually works to still correctly detect and display the colour and progress bar, the docs say that the [default is `auto`](https://doc.rust-lang.org/cargo/reference/config.html#termprogresswhen) and yet it does not work when we explicitly set it. :shrug: 

We could go a step further here and pass in the verbosity to this function so that it only sets the `TERM_PROGRESS` vars when it is not `Verbosity::Quiet`